test/e2e/snaps/test-snap-manageAccount.spec.js

```diff
@@ -68,10 +68,10 @@ describe('Test Snap Account', function () {
           tag: 'button',
         });
 
-        await driver.switchToWindowWithTitle(
-          'SSK - Snap Simple Keyring',
-          windowHandles,
-        );
+        // switch to snap dapp
+        const dappPage = windowHandles[1];
+        await driver.switchToWindow(dappPage);
+        await driver.delay(1000);
```

Comment:
Take a look if really required

Reply:
Removed in 9a99e78e061a45c49525b4e2ed7a7ec7245d74d5